addon-test-support/ember-test-helpers/wait.js

```diff
@@ -1,7 +1,46 @@
 export {
-  default,
   _setupAJAXHooks,
   _setupPromiseListeners,
   _teardownAJAXHooks,
   _teardownPromiseListeners,
 } from '@ember/test-helpers/settled';
+
+import { waitUntil, getSettledState } from '@ember/test-helpers';
+
+/**
+  Returns a promise that resolves when in a settled state (see `isSettled` for
+  a definition of "settled state").
+
+  @private
+  @deprecated
+  @returns {Promise<void>} resolves when settled
+*/
+export default function settled() {
+  let options = arguments[0];
```

Comment:
we should use `function wait(options = {}) {` instead

Reply:
kk, seems good

Reply:
Hmm, actually its a tad different.  The old 0.6.x `wait` code avoided errors when something like a number was passed in, if I use default arguments here it will only correct for `undefined` specifically...

Reply:
sure, the logic to handle non-objects would likely still need to exists, but I think putting `options = {}` in would still be good for documentation purposes

Reply:
updated